client/src/components/Instructions/index.js

```diff
@@ -11,27 +11,7 @@ export default class Instructions extends Component {
         <h2> Build your first app with ZepKit </h2>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            1. In another terminal, start your local blockchain with ganache in a new terminal.
-          </div>
-          <div className={styles.code}>
-            <code>
-              ganache-cli --secure -u 0 -u 1 -u 2 --deterministic
```

Comment:
Not sure if I'd remove this from the instructions. Let's assume that no user will ever read the README.

Reply:
I am not removing it from instructions. I moved it to the right place. In order to see this instruction ganache-cli has to be running so what is why I moved it to other page which is in the same file at line 155.